mriqc/bin/mriqc_run.py

```diff
@@ -201,8 +215,11 @@ def main():
     log_dir = op.join(settings['output_dir'], 'logs')
 
     analysis_levels = opts.analysis_level
-    if opts.participant_label is None:
+
+    # In case direct file path is given (instead of bids dir) we do not want group level
+    if (opts.participant_label is None) & opts.T1w_file is False & opts.T2w_file is False & opts.bold_file is False :
```

Comment:
I do not understand why it should not be modify ...
previously, if no participant_label is given, the group analysis is performed.
So now there is an other case where we do not want group analysis: when T1w_file is given as argument.
sound logical to me ... non ?